src/NumoBundle/Resources/views/site/member.html.twig

```diff
@@ -35,28 +35,26 @@
                 <div class="col-lg-7">
                     <div class="row">
                         <div class="col-lg-offset-0 col-xs-offset-4 col-lg-5">
-                            {% if member.roles.0 == 'ROLE_ADHERENT' %}
+                            {% if member.granted('ROLE_ADHERENT') %}
```

Comment:
j'avais un truc du genre         {% for role in user.getRoles() if role=='ROLE_ADMIN' %}
mais si ta soluce fonctionne, c'est encore mieux, GG ;-)